core/templates/dev/head/app.js

```diff
@@ -471,7 +471,8 @@ oppia.factory('oppiaDatetimeFormatter', ['$filter', function($filter) {
       var date = new Date(millisSinceEpoch);
       if (date.toLocaleDateString() === new Date().toLocaleDateString()) {
         // The replace function removes 'seconds' from the time returned.
-        return date.toLocaleTimeString().replace(/:\d\d /, ' ');
+        return date.toLocaleTimeString('en-US',
```

Comment:
Hey @seanlip, I think I found a way where I won't have to use a specific locale. The only thing is in chrome it will show time as "11:13 am" while in safari and firefox as "11:13 AM". Should I make that change ?